pkg/provisioner/controller.go

```diff
@@ -53,24 +52,26 @@ type Controller struct {
 	obHasSynced  cache.InformerSynced
 	queue        workqueue.RateLimitingInterface
 
-	provisioner     api.Provisioner
-	provisionerName string
+	provisioner        api.Provisioner
+	provisionerName    string
+	provisionerVersion string
 }
 
 var _ controller = &Controller{}
 
-func NewController(provisionerName string, provisioner api.Provisioner, clientset kubernetes.Interface, crdClientSet versioned.Interface, obcInformer informers.ObjectBucketClaimInformer, obInformer informers.ObjectBucketInformer) *Controller {
+func NewController(provisionerName string, operatorVersion string, provisioner api.Provisioner, clientset kubernetes.Interface, crdClientSet versioned.Interface, obcInformer informers.ObjectBucketClaimInformer, obInformer informers.ObjectBucketInformer) *Controller {
```

Comment:
This is a breaking api change. Is there a way to make version optional? Otherwise don't we need to bump the lib version?

Reply:
Where is the lib version?